tests/integration/sms/contacts_interaction_test.js

```diff
@@ -0,0 +1,115 @@
+/* global require, marionette, setup, suite, test */
+'use strict';
+
+var assert = require('chai').assert;
+
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var FromApp = require(appRoot + '/shared/test/integration/require_from_app');
+
+var Messages = FromApp.require('sms', 'lib/messages');
```

Comment:
not related to this patch, just feedback from the user of your lib :) I'd rather have ```fromApp('sms').require('lib/messages')```;